src/bundle/Controller/ContentViewController.php

```diff
@@ -59,6 +76,9 @@ public function locationViewAction(ContentView $view)
         $this->supplyPathLocations($view);
         $this->supplyContentType($view);
         $this->supplyContentActionForms($view);
+        $page = $request->query->get('page');
+        $this->routeName = $request->get('_route');
```

Comment:
This should not be a property of the class. Pass request to the `supplyDraftPagination()` method and retrieve a param in there.